server/src/main/java/org/elasticsearch/index/query/TermsSetQueryBuilder.java

```diff
@@ -74,6 +76,7 @@ public TermsSetQueryBuilder(StreamInput in) throws IOException {
         this.values = (List<?>) in.readGenericValue();
         this.minimumShouldMatchField = in.readOptionalString();
         this.minimumShouldMatchScript = in.readOptionalWriteable(Script::new);
+        this.minimumShouldMatch = in.readOptionalString();
```

Comment:
This needs to handle bwc serialization. 

There are multiple examples around the code base. 

Something like:
```
if (in.getTransportVersion().onOrAfter(TransportVersion.V_8_500_019)) {
..setvalue
}
```
^ Where you create the next transport version type in that class with a UUID